project/Dependencies.scala

```diff
@@ -108,8 +108,10 @@ object Dependencies {
 
   val accessContextManager = "com.google.apis" % "google-api-services-accesscontextmanager" % "v1beta-rev55-1.25.0"
 
-  val workspaceManager = excludeGuavaJDK5("bio.terra" % "workspace-manager-client" % "0.13.0-SNAPSHOT")
-  val dataRepo = excludeGuavaJDK5("bio.terra" % "datarepo-client" % "1.0.44-SNAPSHOT")
+  def swaggerClientExcludes(m: ModuleID): ModuleID = m.exclude("jakarta.activation", "jakarta.activation-api")
+
+  val workspaceManager = swaggerClientExcludes("bio.terra" % "workspace-manager-client" % "0.13.0-SNAPSHOT")
+  val dataRepo = swaggerClientExcludes("bio.terra" % "datarepo-client" % "1.0.44-SNAPSHOT")
```

Comment:
@kshakir @davidangb @asingh7115 Preferable?

Reply:
much better, glad this is working via excludes!

nit, optional: rename `swaggerClientExcludes` to e.g. `excludeJakartaActivation`. If it's only excluding one dependency, make it obvious in the name, similar to `excludeGuavaJDK5`. If it's a bundle of excludes, name it after who uses it, similar to `workbenchGoogleExcludes`.

lastly: do these no longer need to exclude guava-jdk5?

Reply:
They don't actually include guava-jdk5, that's not included in anything vaguely recent. I went with _why_ they have the exclusion over _what_ is being excluded, but I could see inverting it...

Reply:
renamed!